thx/config.py

```diff
@@ -132,7 +132,10 @@ def load_config(path: Optional[Path] = None) -> Config:
         return Config(root=path)
 
     content = pyproject.read_text()
-    data = tomli.loads(content).get("tool", {}).get("thx", {})
+    try:
+        data = tomli.loads(content).get("tool", {}).get("thx", {})
+    except tomli.TOMLDecodeError as error:
+        raise ConfigError(f"failure parsing pyproject.toml: {str(error)}") from error
```

Comment:
Maybe including the actual path could avoid confusion?

```suggestion
        raise ConfigError(f"failure parsing {pyproject}: {error}") from error
```